optimum/intel/openvino/modeling_decoder.py

```diff
@@ -249,6 +251,67 @@ class OVModelForCausalLM(OVBaseDecoderModel, GenerationMixin):
     export_feature = "text-generation"
     auto_model_class = AutoModelForCausalLM
 
+    def __init__(
+        self,
+        model: openvino.runtime.Model,
+        config: PretrainedConfig = None,
+        device: str = "CPU",
+        dynamic_shapes: bool = True,
+        ov_config: Optional[Dict[str, str]] = None,
+        model_save_dir: Optional[Union[str, Path, TemporaryDirectory]] = None,
+        **kwargs,
+    ):
+        model = self._try_modify_pastkv_to_lowprecision(model, ov_config, device, **kwargs)
```

Comment:
It could make sense to have this directly in the `__init__()` of `OVBaseDecoderModel` so that it can be reused for every architecture with a decoder (and pkv) what do you think ?

Reply:
Good idea, updated.